CONTRIBUTING.md

```diff
@@ -4,41 +4,52 @@
 [pr]: https://github.com/actions/dependency-review-action/compare
 [code-of-conduct]: CODE_OF_CONDUCT.md
 
-Hi there! We're thrilled that you'd like to contribute to this project. Your help is essential for keeping it great.
+Hi there! We're thrilled that you'd like to contribute to this project.
 
-Contributions to this project are
-[released](https://help.github.com/articles/github-terms-of-service/#6-contributions-under-repository-license)
-to the public under the [project's open source license](LICENSE).
+Contributions to this project are [released](https://help.github.com/articles/github-terms-of-service/#6-contributions-under-repository-license) to the public under the [project's open source license](LICENSE).
 
-Please note that this project is released with a [Contributor Code of
-Conduct][code-of-conduct]. By participating in this project you agree
-to abide by its terms.
+Please note that this project is released with a [Contributor Code of Conduct][code-of-conduct]. By participating in this project you agree to abide by its terms.
 
-### How it works
+## Bug reports and other issues
 
-This Action makes an authenticated query to the Dependency Graph Diff
-API endpoint (`GET /repos/{owner}/{repo}/dependency-graph/compare/{basehead}`)
-to find out the set of added and removed dependencies for each manifest.
+If you've encountered a problem, please let us know by [submitting an issue](https://github.com/actions/dependency-review-action/issues/new)!
 
-### Bootstrapping the project
+## Enhancements and feature requests
 
-```
-git clone https://github.com/actions/dependency-review-action.git
-cd dependency-review-action
-npm install
-```
+If you've got an idea for a new feature, please submit as [an issue](https://github.com/actions/dependency-review-action/issues/new) so that the community can see it, and we can discuss it there. We may not be able to respond to every single issue, but will make a best effort!
 
-### Running the tests
+If you'd like to make a contribution yourself, we ask that before significant effort is put into code changes, that we have agreement that the change aligns with our strategy for the action. Since GitHub is ultimately the primary maintainer of this action, we want to make sure that contributions are high quality, and that they maintain consistency with the rest of the action's behavior.
```

Comment:
```suggestion
If you'd like to make a contribution yourself, we ask that before significant effort is put into code changes, that we have agreement that the change aligns with our strategy for the action. Since this is a verified Action owned by GitHub we want to make sure that contributions are high quality, and that they maintain consistency with the rest of the action's behavior.
```

I think it's important to underscore verification here, this code has a high watermark in terms of trust.